build/html/docs/api_description.md

```diff
@@ -0,0 +1,136 @@
+ 
+# Introduction
+Welcome to HPE GreenLake for bare metal APIs. 
+
+The HPE GreenLake for bare metal is included as part of the HPE GreenLake for Private Cloud
+Enterprise service, and it is sometimes also called Bare Metal as a Service or BMaaS.
+
+This document describes the HPE GreenLake for bare metal APIs protocol and available endpoints.
+
+The BMaaS API is built on HTTP and is RESTful. It has predictable resource URLs. It retuns HTTP 
+response codes to indicate errors. It also accepts and returns JSON in the HTTP body. You can 
+use your favorite HTTP/REST library for your promming language to use HPE GreenLake for Private 
+Cloud Enterprise bare metal APIs.
+
+The APIs provides access to bare metal service within a single project context. Clients can create 
+a fully-provisioned hosts, storage volumes, and project-specific private networks in an isolated 
+project environment.  Clients can also access information about available services and resources 
+through the available-resources API.  This API provides detailed information about the OS imaging 
+options, the machine size options, the storage volume options, data center locations,and such that 
+are needed when creating hosts and volumes.
+
+The API specification references a few resources by the old name, which differs from the name 
```

Comment:
Since this will be the first version provided externally, should we be using something other than "old" to explain the mapping?

Reply:
does 'obsolete' fit better?

Reply:
Yes -- obsolete should be good to use.

Reply:
used 'deprecated' instead of 'old'/'obsolete'.